app/graphql/mutations/users/update.rb

```diff
@@ -25,6 +29,12 @@ def resolve(user_id:, mfa: nil, **params)
 
         return { user: nil, errors: [create_message_error('Invalid user')] } if user.nil?
 
+        if params[:password] != params[:password_repeat]
+          return { user: nil, errors: [create_message_error('Invalid password repeat')] }
+        end
+
+        params.delete(:password_repeat)
```

Comment:
This can be combined
```suggestion
        if params[:password] != params.delete(:password_repeat)
          return { user: nil, errors: [create_message_error('Invalid password repeat')] }
        end
```